src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -401,7 +402,26 @@ public static Config BaseConfig
             ActorSystem system,
             ImmutableList<RoleName> roles,
             Func<RoleName, ImmutableList<string>> deployments)
-            : base(new XunitAssertions(), system)
+            : this(myself, system, null, roles, deployments)
+        {
+        }
+
+        protected MultiNodeSpec(
+            RoleName myself,
+            ActorSystemSetup setup,
+            ImmutableList<RoleName> roles,
+            Func<RoleName, ImmutableList<string>> deployments)
+            : this(myself, null, setup, roles, deployments)
+        {
+        }
+
+        private MultiNodeSpec(
+            RoleName myself,
+            ActorSystem system,
+            ActorSystemSetup setup,
```

Comment:
Got it - so this is really an option - you can pass in either a full-blown `ActorSystem` or an `ActorSystemSetup` but only one of them gets used - the former if it's not `null`.-

Reply:
Yes, exactly.